src/gpodder/gtkui/shownotes.py

```diff
@@ -154,3 +170,105 @@ def activate_links(self):
             target = next((url for start, end, url in self.hyperlinks if start < pos < end), None)
             if target is not None:
                 util.open_website(target)
+
+
+class gPodderShownotesHTML(gPodderShownotes):
+    def init(self):
+        self.html_view = WebKit2.WebView()
+        self.html_view.set_property('expand', True)
+        self.html_view.connect('mouse-target-changed', self.on_mouse_over)
+        self.html_view.connect('context-menu', self.on_context_menu)
+        self.html_view.connect('decide-policy', self.on_decide_policy)
+        self.header = Gtk.Label.new()
+        self.header.set_halign(Gtk.Align.START)
+        self.header.set_valign(Gtk.Align.START)
+        self.header.set_property('margin', 10)
+        self.status = Gtk.Label.new()
+        self.status.set_halign(Gtk.Align.START)
+        self.status.set_valign(Gtk.Align.END)
+        self.set_status(None)
+        grid = Gtk.Grid()
+        grid.attach(self.header, 0, 0, 1, 1)
+        grid.attach(self.html_view, 0, 1, 1, 1)
+        grid.attach(self.status, 0, 2, 1, 1)
+        return grid
+
+    def update(self, heading, subheading, episode):
+        tmpl = '<span size="x-large" font_weight="bold">%s</span>\n' \
+              +'<span size="medium">%s</span>'
+        self.header.set_markup(tmpl % (heading, subheading))
```

Comment:
Heading and subheading might need `cgi.escape()` or similar except if that's already done somewhere on the caller side (didn't check).